src/bin/mnemosyne/fat/layout/src/collections/array.rs

```diff
@@ -0,0 +1,93 @@
+use crate::{io::*, *};
+
+// specialization o_O
+
+// impl<const N: usize> Decode for [u8; N] {
```

Comment:
Don't check in commented out code.  We use a version control system so if you need old code, you can always restore it later.

Reply:
Sorry I copied it over from Leo's repo since it's private, I didn't check it too closely, will do.